README.md

```diff
@@ -191,22 +191,41 @@ py.test
 #### The test data subset
 If you add new tables to the data, you'll need to generate a new subset for testing. We use this nifty subsetting tool: [rdbms-subsetter](https://github.com/18F/rdbms-subsetter).
 
-To build a new test subset, use the `build_test` invoke task:
+To build a new test subset, first create a new database for yourself:
+
+```
+createdb cfdm_new_subset
```

Comment:
Ah okay, which `invoke` command are you referring to?  The `build_test` one or the `dump` one?  If you're referring to the `build_test` one, it appears that needs a connection string to a database for both the source and destination and not a file path.

Could you please paste in the last one you ran (or tried to run) when you have a moment so I can see?  I think I'm either missing something or just generally confused and I want to make sure I'm on the same page as you.